fixbaserom.py

```diff
@@ -68,11 +73,19 @@ def get_str_hash(byte_array):
 print("Patching header...")
 fileContent[0x3E] = 0x50
 
+for i in range(0x35CF000, len(fileContent)):
+    fileContent[i] = 0xFF
```

Comment:
What's the advantage of doing this?

Reply:
the whole goal with this PR is make it easier for people to use whatever equivalent rom they have

there are roms floating around that have 0x00 padding instead of 0xFF at the end of the rom

(the rom "elf" ends at 0x35CF000, cf map file `0x00000000035cf000                _RomEnd = _RomSize`)